admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -93,7 +93,9 @@ object DelegateApiHandler {
   sealed trait JsonDelegateTree
   object JsonDelegateTree {
     case class Empty(path: Path, dentry: Option[Dentry]) extends JsonDelegateTree
-    case class Fail(path: Path, dentry: Option[Dentry]) extends JsonDelegateTree
+    case class Fail(dentry: Option[Dentry]) extends JsonDelegateTree {
+      val path = "!"
```

Comment:
How do you feel about using the descriptive paths like `/$/fail` instead?  I find these symbols to be really hard to keep track of mentally.